backend/fms_core/viewsets/sample.py

```diff
@@ -324,14 +324,38 @@ def list_export(self, _request):
         serialized_data = self.fetch_export_data()
         return Response(serialized_data)
 
+    @action(detail=False, methods=["get"])
+    def list_export_metadata(self, _request):
+        self.queryset = self.filter_queryset(self.get_queryset())
+        serialized_data = self.fetch_export_metadata()
```

Comment:
We need to think about the use cases ... Will people want them by sample, will they want the pools metadata broken down, should we prevent pools to be included ... I would think people will want to extract the meta data for any sample even if they are pooled ... so we would need to return them broken down by biosample/derived sample...